index.html

```diff
@@ -211,5 +211,23 @@ <h1>About me</h1>
         </div>
       </div>
     </section>
+      <div class="contact-me">
+        <h1 class="contact-heading">
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h1>

```

Comment:
- [ ] The heading of the contact section should follow the heading structure and hierarchy of the page and should not be an `h1`. There should be no more than one `h1` tag on the page and it should be the main title. Kindly change that to an `h2` tag and take a look [at this article](https://www.reliablesoft.net/h1-tag/) for more information. 👍